raiden/blockchain_events_handler.py

```diff
@@ -215,6 +216,27 @@ def handle_channel_closed(raiden, event, current_block_number):
         raiden.handle_state_change(channel_closed, current_block_number)
 
 
+def handle_channel_update_transfer(raiden, event, current_block_number):
+    token_network_identifier = event.originating_contract
+    data = event.event_data
+    channel_identifier = data['channel_identifier']
+
+    channel_state = views.get_channelstate_by_token_network_identifier(
+        views.state_from_raiden(raiden),
+        token_network_identifier,
+        channel_identifier,
+    )
+
+    if channel_state:
+        channel_transfer_updated = ContractReceiveUpdateTransfer(
+            transaction_from=data['closing_participant'],
```

Comment:
what about the address of the transaction sender?

Reply:
It's not at the event. As agreed in the office we should store the transaction hash in the future.